utils/findutils.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="http://repo.roscidus.com/utils/findutils" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>FindUtils</name>
+  <summary xml:lang="en">FindUtils: find and then operate on files</summary>
+  <description xml:lang="en">`find' is a program which searches a directory tree to find a file or group of files. It walks the directory tree and reports all occurences of a file matching the user's specifications. 
+
+`locate' scans one or more databases of filenames and displays any matches. 
+
+`xargs' builds and executes command lines by gathering together arguments it reads on the standard input. Most often, these arguments are lists of file names generated by `find'
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.png" type="image/png"/>
+  <category>System</category>
+  <homepage>http://gnuwin32.sourceforge.net/packages/findutils.htm</homepage>
+  <needs-terminal/>
+  <implementation arch="Windows-i486" id="sha1new=6996e25456413b9e8b43c4b336e0c05d5e2509d9" langs="be ca da de el es et fi fr ga gl hr hu id it ja ko ms nl pl pt pt_BR ro ru sk sl sr sv tr vi zh_CN" license="GPL v2 (GNU General Public License)" released="2005-04-18" version="4.2.20-2-3">
```

Comment:
I'd suggest removing the `langs` attribute as long as there are no other implementations with a different set of languages for 0install to potentially choose.

Reply:
I have added an alternative implementation from ezwinports that does not include langs

>  This is a Windows port of GNU Findutils 4.2.30.  It includes all the
  binaries in the original distribution, even those, such as code.exe,
  frcode.exe, and bigram.exe that you are not supposed to need.  I
  made this port because the GnuWin32 port of v4.2.20 had several
  grave problems: find.exe was abysmally slow, xargs.exe didn't work
  at all, and neither did locate.exe.  This port includes updatedb.bat,
  which is a stripped-down replacement for the original updatedb Unix
  shell script.  Please note that xargs.exe in this distribution has
  some issues that cause it to fail sometimes.  But find.exe and
  locate.exe work flawlessly for me for several years.

>  This fifth upload solves a bug whereby 'find' would sometimes
  infloop when some of the directories it traverses are symlinks to
  other directories.  The solution is to never descend into a
  directory that is a symlink (i.e., this port always behaves as if
  the -P option was specified).

>  As an additional bonus, this port includes 64-bit binariesin a
  separate findutils-4.2.30-4-w64-bin zip file, prepared by Erwin
  Waterlander &lt;waterlan@xs4all.nl&gt;.  If you are running a 64-bit
  Windows version, you may wish to use these 64-bit executables, as
  they are about 5 times faster than the 32-bit ones.